### PR TITLE
alternative way to show hidden options

### PR DESCRIPTION
**Pull request by borg323**

### CODE REVIEW

File: src/utils/optionsparser.cc [hunk 1]
```diff
@@ -120,6 +120,9 @@ bool OptionsParser::ProcessAllFlags() {
 
 bool OptionsParser::ProcessFlags(const std::vector<std::string>& args) {
   auto show_help = false;
+  if (CommandLine::BinaryName().find("expert") != std::string::npos) {
```
Comment (mooskagh) [inline src/utils/optionsparser.cc:123]: Feel free also to pick some funnier/mysterious substring for that. :)

Comment (fohristiwhirl) [reply src/utils/optionsparser.cc:123]: "galaxybrain"

Comment (borg323) [reply src/utils/optionsparser.cc:123]: I was leaning towards "pro" for a more ironic approach.

Comment (mooskagh) [reply src/utils/optionsparser.cc:123]: Have no objections to either of options. Or `hackerman` to get "pro" irony even further. :)

Comment (Naphthalin) [reply src/utils/optionsparser.cc:123]: I propose `sudo`, as changing hidden options should require knowledge about the inner workings and be done on your own risk.

**Pull request merged**